File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.java
class Solution {
    public String solution(String s) {
        char[] str = s.toCharArray();
        int len = str.length;
        
        if (len % 2 == 0) {
            return String.valueOf(str[len / 2 - 1]) + str[len / 2];
        } else {
            return String.valueOf(str[len / 2]);
        }
    }
}

File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.py
def solution(s):
    mid = len(s) // 2
    # 파이썬 삼항연산자 : 값1 if 조건 else 값2
    return s[mid] if len(s) % 2 == 1 else s[mid - 1:mid + 1] 
